core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -705,7 +707,8 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     OneTimeConsumeResources("Instantly consumes [amount] [stockpiledResource]", UniqueTarget.Triggerable),
     OneTimeProvideResources("Instantly provides [amount] [stockpiledResource]", UniqueTarget.Triggerable),
 
-    OneTimeGainStat("Gain [amount] [stat/resource]", UniqueTarget.Triggerable),
```

Comment:
Please don't override :)

Reply:
The only change there is the text. This is already replaced by `OneTimeProvideResources`, and the trigger already longer checked resources

Reply:
> forget it.

Yeah, it looks weird, but that's just github's diff engine combining stuff unnecessarily